### ISSUE TITLE
Automated checks for groundwater levels

### ISSUE THREAD

**Issue by clau1313** [kartoza/IGRAC-GGIS#253]
"Water depth [from the top of the well]" should always be less or equal that "total depth (from Top of Borehole)". Likewise, "water depth [from ground surface]" should always be less or equal that "total depth (from Ground Surface Level ASL)". In other words, the water level cannot be deeper than the total depth of the well. Same when considering water level elevation a.m.s.l.

Moreover, when no unit is selected for a measurement, the system assigns m2/day, which should not be like that but the option by default should be "m" instead.

In conclusion, this value should not be allowed:

![image](https://user-images.githubusercontent.com/36033176/102496817-2f5fe900-4078-11eb-9d29-3c0c720e12a3.png)

@meomancer
